testsuite/smoke-test/src/client.rs

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::test_utils::{
+    compare_balances, load_libra_root_storage, setup_swarm_and_client_proxy, test_smoke_script,
+};
+use libra_config::config::NodeConfig;
+use libra_trace::trace::trace_node;
+use libra_types::{
+    account_address::AccountAddress, account_config::testnet_dd_account_address,
+    ledger_info::LedgerInfo, waypoint::Waypoint,
+};
+
+#[test]
+fn smoke_test_multi_node() {
```

Comment:
why this file is called `client`?

Reply:
Yeah, this file is a bit of a mixed bag of random tests. The only reason I called them client is because all the tests seem to be heavily focussed around performing client proxy operations and testing functionality (e.g., testing client sends, client account creation, client waypoints, client balance fetching, client event tracing etc.) If you have a better suggestion for how to group them, please let me know 😄 

One possibility would be to have several finer grained files:
1. `basic.rs` for basic functionality (e.g., fault tolerance and restartability)
2. `client.rs` for the client waypoint test and client trace support test.
3. `accounts_and_balances.rs` for tests that do basic things like create accounts, transfer funds and check balances.

Reply:
maybe 1&3 can combine together